libs/gl-client/Cargo.toml

```diff
@@ -45,7 +45,7 @@ vls-protocol-signer = { workspace = true }
 vls-protocol = { workspace = true }
 serde_json = "^1.0"
 thiserror = "1"
-cln-grpc = { workspace = true }
+cln-grpc = { version = "0.1.8", workspace = true }
```

Comment:
This pin needs to be done on the root level, otherwise we end up with divergent versions between components again.

https://github.com/Blockstream/greenlight/blob/292edcf2bf3625174707a59a8a553d9159b890b2/Cargo.toml#L25